enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -95,263 +112,214 @@ This is an example of an OKD cluster hosting solution where several organization
-  </label>
-</source>
- 
-# INGRESS
-<label @INGRESS>
-  ... #current @INGRESS config
-
-    <match output_ops_tag journal.** system.var.log** mux.ops audit.log**   %OCP_FLUENTD_TAGS%>
-      @type relabel
-      @label @_LOGS_INFRA
-    </match>
-    <match kubernetes.**>
-      @type relabel
-      @label @_LOGS_APP
-    </match>
-</label>
-
-<label @_LOGS_APP>
-  <match **>
-     @type copy
-     <store>
-        @type relabel
-        @label @CONTAINER_LOGS
-     </store>
-  </match>
-</label>
-
-<label @_LOGS_INFRA>
-  <match **>
-     @type copy
-     <store>
-        @type relabel
-        @label @INFRA_LOGS
-     </store>
-  </match>
-</label>
-
-# match outputs are generated based on defined pipeline endpoints
-<label @CONTAINER_LOGS>
-  <match **>
-     @type copy
-     #store section for each endpoint type
-     <store>
-       @type relabel
-       @label @SECUREFORWARD_OFFCLUSTER
-     </store>
-     <store>
-       @type relabel
-       @label @ELASTICSEARCH
-     </store>
- </match>
-</label>
-
-<label @CONTAINER_LOGS>
-  <match **>
-     @type copy
-     #store section for each endpoint type
-     <store>
-       @type relabel
-       @label @ELASTICSEARCH_INSECURE
-     </store>
- </match>
-</label>
- 
-#specific pipeline endpoint
-<label @ELASTICSEARCH>
-  <match retry_elasticsearch>
-     @type copy
-     <store>
-       @type elasticsearch
-       @id elasticsearch
-       client_key /var/run/ocp-collector/secrets/elasticsearch/key
-       client_cert /var/run/ocp-collector/secrets/elasticsearch/cert
-       ca_file /var/run/ocp-collector/secrets/elasticsearch/cacert
-       <buffer>
-		@type file
-		path '/var/lib/fluentd/elasticsearch'
-       </buffer>
-     </store>
-  </match>
-  <match **>
-     @type copy
-     <store>
-       @type elasticsearch
-       @id elasticsearch
-       client_key /var/run/ocp-collector/secrets/elasticsearch/key
-       client_cert /var/run/ocp-collector/secrets/elasticsearch/cert
-       ca_file /var/run/ocp-collector/secrets/elasticsearch/cacert
-       retry_tag "retry_elasticsearch"
-     </store>
- </match>
-</label>
-<label @SECUREFORWARD_OFFCLUSTER>
-   ...
-</label>
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogForwarder"
+spec:
+  outputs:
+   - name: MyLogs
+     type: Syslog
+     syslog: {Facility: Local0}
+     url: localstore.example.com:9200
+  pipelines:
+   - inputs: [Infrastructure, Application, Audit]
+     outputs: [MyLogs]
 ```
 
-Following is a diagram which identifies the workflow of a message through fluentd as captured by the above configuration.
+#### As a cluster administrator, I want to clearly separate where the logging stack forwards infrastructure and/or audit related logs.
 
-![https://docs.google.com/drawings/d/18FD4c1pBz4e8eJIH37yUwM40ew69g8h1cqPEfOLsuyY/edit?usp=sharing](images/log-forwarding-message-flow.png)
+```
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogForwarder"
+metadata:
+  namespace: cluster-logging # Global forwarder
```

Comment:
I think we discussed this being a cluster scoped resource meaning it wont have a namespace. Also, kind of goes along with the naming convention

Reply:
Correct, also an accidental leftover.